packages/chevrotain/src/parse/parser/traits/looksahead.ts

```diff
@@ -52,6 +46,10 @@ export class LooksAhead {
       ? (config.maxLookahead as number) // assumes end user provides the correct config value/type
       : DEFAULT_PARSER_CONFIG.maxLookahead
 
+    this.lookaheadStrategy = has(config, "lookaheadStrategy")
+      ? (config.lookaheadStrategy as ILookaheadStrategy) // assumes end user provides the correct config value/type
+      : DEFAULT_PARSER_CONFIG.lookaheadStrategy
```

Comment:
Does it make sense to add the `maxLookahead` state to the strategy at the constructor invocation?